src/app/banners/BannerWrapper.tsx

```diff
@@ -17,11 +17,13 @@ const BannerWrapper = (): JSX.Element => {
   const userPlan = plan.subscription?.type;
   const isNewAccount = useAppSelector(userSelectors.hasSignedToday);
   const shouldShowBanner = userPlan === 'free' && !getCookie(SHOW_BANNER_COOKIE_NAME);
-  const expireDate = new Date('2024-01-01');
+  const expireDate = new Date('2024-01-10T23:59:59.000Z');
```

Comment:
This new banner expires on the 10th of this month?

Reply:
Yes, I have to remove this banner on 10th and add another one for a new soft sale.